### PR TITLE
feat: added Wishlist and cart

### PR DESCRIPTION
**Pull request by Asuraking1n**
added Wishlist and cart

### CODE REVIEW

File: src/components/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    width: 100%;
+    background: url(../../../public/images/shop-header-bg.jpg) no-repeat center center/cover;
+}
+
+.cart-container-heading{
+    font-weight: 600;
+    font-style: normal;
+    color: #a8e8e2;
+    font-size: 4.375rem;
+    line-height: 70px;
```
Comment (rahul31agrawal) [inline src/components/cart/cart.css:12]: use rems instead of px

Comment (Asuraking1n) [reply src/components/cart/cart.css:12]: No i don't want it to change according to size...that's why it's in px

**Review by rahul31agrawal — APPROVED** [review]
Hi Nishant,
Your ecomm website looks amazing.
I would suggest you to choose different font family for this & also the fonts for "unit price" & stock status is not visible properly, so it would be great if you show them in black color.
please ignore if my suggestion seem irrelavant.

Thanks

**Comment by Asuraking1n** [comment]
> Hi Nishant, Your ecomm website looks amazing. I would suggest you to choose different font family for this & also the fonts for "unit price" & stock status is not visible properly, so it would be great if you show them in black color. please ignore if my suggestion seem irrelavant.
> 
> Thanks

Sure...I will look into it

**Pull request merged**